### PR TITLE
PR as discussed in https://github.com/prophesee-ai/prophesee_ros_wrapper/pull/7

### PR DESCRIPTION
**Pull request by max11gen**
As discussed with @nlyubova in [this PR](https://github.com/prophesee-ai/prophesee_ros_wrapper/pull/7), here is my PR. I deleted the prophesee_viewer since I didn't implement a stereo viewer yet and the viewer of the prophesee_ros_driver package can still be used for now. When I find some time, I'll implement a stereo viewer and make another PR.

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -2,10 +2,11 @@
 <package format="2">
   <name>prophesee_ros_stereo_driver</name>
   <version>0.0.2</version>
-  <description>Publisher and Viewer for Prophesee Event-Based Stereo Sensors</description>
+  <description>Stereo Publisher and Viewer for Prophesee Event-Based Sensors</description>
 
   <maintainer email="nlyubova@prophesee.ai">Natalia Lyubova</maintainer>
   <author email="natalia.lyubova@prophesee.ai">Natalia Lyubova</author>
+  <author email="egor.makhov@mail.polimi.it">Egor Makhov</author>
```
Comment (nlyubova) [inline package.xml]: could you please remove this line and add instead of it your name ? as you did the code

**Review by nlyubova — CHANGES_REQUESTED** [review]
Thanks for your PR !
I've passed the code very quickly and I had only one comment for now. Then, we'll test it with stereo and let you know.

File: package.xml (hunk 1 above)
Comment (max11gen) [reply package.xml]: Done.

File: include/prophesee_ros_stereo_driver/prophesee_ros_stereo_publisher.h [hunk 2]
```diff
@@ -0,0 +1,130 @@
+/*******************************************************************
+ * File : prophesee_ros_publisher.h                                *
+ *                                                                 *
+ * Copyright: (c) 2015-2019 Prophesee                              *
+ *******************************************************************/
+
+#ifndef STEREO_PUBLISHER_H_
+#define STEREO_ROS_PUBLISHER_H_
```
Comment (atonda-psee) [inline include/prophesee_ros_stereo_driver/prophesee_ros_stereo_publisher.h]: some inconsistency here.
should be #ifndef STEREO_ROS_PUBLISHER_H_

**Review by atonda-psee — CHANGES_REQUESTED** [review]

File: include/prophesee_ros_stereo_driver/prophesee_ros_stereo_publisher.h (hunk 2 above)
Comment (max11gen) [reply include/prophesee_ros_stereo_driver/prophesee_ros_stereo_publisher.h]: Fixed, thanks.

File: launch/prophesee_viewer.launch [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<launch>
+
+  <node pkg="prophesee_ros_driver" type="prophesee_ros_viewer" name="prophesee_ros_viewer" respawn="false" output="screen">
+    <param name="camera_name" value="camera" />
```
Comment (nlyubova) [inline launch/prophesee_viewer.launch]: please, change here 
value="camera" /> to value="camera_left" />

File: launch/prophesee_publisher.launch [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<launch>
```
Comment (nlyubova) [inline launch/prophesee_publisher.launch]: please, remove this file

File: launch/prophesee_stereo_publisher.launch [hunk 5]
```diff
@@ -1,19 +1,20 @@
...
+    <param name="right_camera_id" value="00002402" />
+    <param name="master_left" value="true" />
     <param name="publish_cd" value="true" />
     <param name="publish_graylevels" value="false" />
-    <param name="publish_imu" value="true" />
+    <param name="publish_imu" value="false" />
 
     <!-- Path to the file with the camera settings -->
-    <param name="bias_file" value="" />
+    <param name="bias_file" value="/home/neuroreact/catkin_ws/src/neuromorphic_stereo/biases/gen3_EM_sens40.bias" />
```
Comment (nlyubova) [inline launch/prophesee_stereo_publisher.launch]: Could you please remove this value by default?

**Review by nlyubova — CHANGES_REQUESTED** [review]
Hi, I have one last comment only and then it's good for me !

**Comment by max11gen** [comment]
Hi @nlyubova,
Sorry for the messy commit history. I had already fixed most of those things you mentioned in a commit on friday (everything except for the default path). Now I had some merging issues because you and me simultaneously changed/deleted/added launch files and I had to rebase and force push. The repo is clean now and I also tested the prophesee_stereo_viewer.launch.

**Comment by nlyubova** [comment]
Thanks for your changes !
It's good for me.
Let me check with my colleague and then we'll merge it

**Pull request merged**